.travis.yml

```diff
@@ -8,6 +8,9 @@ services:
 compiler:
   - gcc
 
+arch:
+  - ppc64le
```

Comment:
With this it will build ppc64le only, no? I believe you need to specify amd64 as well here.